src/index.html

```diff
@@ -11,7 +11,179 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      class="forms"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <!-- Personal Information --------------------- -->
+      <fieldset
+        class="field"
+      >
```

Comment:
```suggestion
      <fieldset class="field">
```
you can write open-tag in 1 line if there 1-2 short attributes